services/QuillLMS/app/workers/canvas_integration/import_teacher_classrooms_students_worker.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module CanvasIntegration
+  class ImportTeacherClassroomsStudentsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: SidekiqQueue::CRITICAL_EXTERNAL
+
+    PUSHER_EVENT = 'canvas-classroom-students-imported'
+    PUSHER_FAILED_EVENT = 'canvas-account-reauthorization-required'
```

Comment:
As with my note on another PR, is there front-end code set up to listen for these Pusher notifications?  (That may be a future thing, and not necessary yet, I just want to make sure that it's on the radar.)

Reply:
Thank you for raising this.  In this case, there isn't canvas frontend code yet.